examples/async_pipeline.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>small typos: "it logic"; "be execute"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-experimental/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>